src/AppBundle/Membership/AdherentEmailSubscription.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace AppBundle\Membership;
+
+final class AdherentEmailSubscription
+{
+    const SUBSCRIBED_EMAILS_MAIN = 'subscribed_emails_main';
+    const SUBSCRIBED_EMAILS_REFERENTS = 'subscribed_emails_referents';
+    const SUBSCRIBED_EMAILS_LOCAL_HOST = 'subscribed_emails_local_host';
+
+    public static function getSubscriptions()
```

Comment:
But it's compiled and cached. A method call is more CPU consumming AFAIK. Anyway it doesn't matter ^^